src/ui.rs

```diff
@@ -55,8 +58,49 @@ pub enum Action<'a> {
     Noop,
 }
 
+/// Actions to perform while in MultiNormal Mode
+#[derive(Clone, Debug)]
+pub enum MultiNormalAction<'a> {
+    /// Rerender screen
+    ReRender,
+    /// Keep listening for input. Update display with current input
+    MoreInput,
+    /// Done getting Input. Repeat the command the specified times
+    Repeat(ProcessAction<'a>),
+    /// Switch back to normal mode
+    SwitchBackNormalMode,
+    /// Cancels input for entering how many times to repeat
+    /// Switches back to Normal mode as well.
+    Cancel,
+    /// Notify to quit out of program
+    Quit,
+    /// Do not respond to event
+    Noop,
+}
+
+impl<'a> From<ProcessAction<'a>> for MultiNormalAction<'a> {
+    fn from(item: ProcessAction<'a>) -> Self {
+        MultiNormalAction::Repeat(item)
+    }
+}
+
+/// Perform an Action `times` times
+#[derive(Clone, Debug)]
+pub struct ProcessAction<'a> {
```

Comment:
If you're just associating two variables, why not use a tuple? Is there a benefit of a `struct` versus a tuple?

Reply:
I went with a struct for the labeled fields (action, and times).
Also being able to implement conversions (such as from ProcessAction -> MultiAction)